src/aat/resources/features/F-029/S-049.td.json

```diff
@@ -0,0 +1,1223 @@
+{
+  "title": "must return successfully a CaseEventTrigger object filtered by case type ID",
+  "_guid_": "S-049",
+  "_extends_": "F-029_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD",
+    "is prepared with valid input parameters",
+    "has the 200 return code"
+  ],
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "headers": {
+      "Transfer-Encoding": "chunked"
+    },
+    "body": {
+      "id": "CREATE",
+      "name": "Create a new case",
+      "description": null,
+      "case_id": null,
+      "case_fields": [
+        {
+          "id": "TextField",
+          "label": "A `Text` field",
+          "hidden": false,
+          "value": null,
+          "metadata": false,
+          "hint_text": "Some generic textual data.",
+          "field_type": {
+            "id": "Text",
+            "type": "Text",
+            "min": null,
+            "max": null,
+            "regular_expression": null,
+            "fixed_list_items": [],
+            "complex_fields": [],
+            "collection_field_type": null
+          },
+          "validation_expr": null,
+          "security_label": "PUBLIC",
+          "order": null,
+          "display_context": "OPTIONAL",
+          "display_context_parameter": null,
+          "show_condition": null,
+          "show_summary_change_option": true,
+          "show_summary_content_option": null,
+          "acls": [
+            {
+              "role": "caseworker-autotest1",
+              "create": true,
+              "read": true,
+              "update": true,
+              "delete": false
+            }
+          ]
+        },
+        {
+          "id": "NumberField",
+          "label": "A `Number` field",
```

Comment:
we have the whole json as test data. What happens if we needed to introduce a new mandatory field? will we have to update all the json test data with that new field?

Reply:
Ideally, as you'd appreciate, before making a change in a feature, we should change the test script/data/config as needed. To reduce the data maintenance issues, fw is providing very flexible and powerful inheritance mechanism to reuse a test data stored in a json inside many other json files. In the case of a same test data repeated in multiple places for any reason, we'll have to adjust all test data, yes.